### PR TITLE
SUBS-619 New Footer and MG Link in Footer

### PR DESCRIPTION
**Pull request by eddieferrer**
* New Footer design for new stack UI desktop resolution with MG link in footer and lend link.

Only includes the new design for new stack desktop resolutions.

Still TODO in upcoming PR's - 

- New stack mobile 
- Minimal legacy reskin 

Screen Shot 2021-02-22 at 3 58 58 PM

### CODE REVIEW

**Review by mcstover — APPROVED** [review]
sooo fresh!

File: src/components/WwwFrame/TheFooter.vue [hunk 1]
```diff
@@ -534,75 +568,45 @@ $footer-separator-color: $light-green;
...
+			padding: 1rem 0.9375rem;
+			text-align: left;
+		}
 
-			ul {
-				li {
-					line-height: 1.6;
-				}
-			}
+		& .large-footer__single-line-links {
```
Comment (ryan-ludwig) [inline src/components/WwwFrame/TheFooter.vue:580]: Can this just be `&__single-line-links`?

File: src/components/WwwFrame/TheFooter.vue [hunk 2]
```diff
@@ -534,75 +568,45 @@ $footer-separator-color: $light-green;
...
 			}
 		}
 
-		.work-with-us {
-			$spacing: 0.5rem;
-
-			ul {
-				overflow: hidden;
-				width: calc(100% + #{$spacing * 2} + 1px);
+		& .large-footer__single-line-links--work-with-us {
```
Comment (ryan-ludwig) [inline src/components/WwwFrame/TheFooter.vue:600]: Can this be `__single-line-links--work-with-us` ?

**Review by ryan-ludwig — APPROVED** [review]
Love the additional contrast!

**Review by emuvente — APPROVED** [review]
👍

**Review by BoulderBrains — APPROVED** [review]

File: src/components/WwwFrame/TheFooter.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/WwwFrame/TheFooter.vue:580]: I tried that but stylelint gave a warning about the order of this vs styling above.  Same with the one below. I tried disabling stylelint for the line with a comment, but that didnt seem to work at all.

**Pull request merged**